docker-compose.yaml

```diff
@@ -7,76 +7,70 @@
 # and only if build args SETTLS and SETPROXY were set during
 # the build of the parent owasp/modsecurity:2.9-apache image.
 
-services:
-  crs-apache:
-    image: owasp/modsecurity-crs:apache
-    ports:
-      - "80:80"
-      # only available if SETTLS was enabled:
-      - "443:443"
-
-    environment:
-      - SERVERNAME=localhost
+version: '3.7'
```

Comment:
The `version` field is only informative. It's removal has been recommended for some time now: https://docs.docker.com/compose/compose-file/04-version-and-name/.